File: leetcode/problems/cpp/1344. Angle Between Hands of a Clock/solve.cpp
#include <algorithm>
#include <cmath>

class Solution {
public:
  double angleClock(int hour, int minutes) {
    double minuteAngle = minutes * 6;
    double hourAngle = (hour % 12) * 30 + minutes / 2.0;
    double diff = std::abs(hourAngle - minuteAngle);
    return std::min(diff, 360 - diff);
  }
};

File: leetcode/problems/java/1344. Angle Between Hands of a Clock/solve.java
class Solution {
    public double angleClock(int hour, int minutes) {
        double minuteAngle = (double) minutes * 6;
        double hourAngle = ((double) hour % 12) * 30 + ((double) minutes / 2);
        double diff = Math.abs(hourAngle - minuteAngle);
        return Math.min(diff, 360 - diff);
    }
}
